src/main/java/xyz/duncanruns/julti/gui/OptionsGUI.java

```diff
@@ -663,5 +666,10 @@ private void onClose() {
             SleepBGUtil.disableLock();
             DoAllFastUtil.doAllFast(minecraftInstance -> minecraftInstance.ensureResettingWindowState(false));
         });
+        try {
+            Julti.isWallAHKOpen();
+        } catch (Exception e) {
+            Julti.log(Level.ERROR, "Failed to detect Wall Macro.");
+        }
```

Comment:
Having to a do a try except here is a little annoying and adds a lot of bloat to onClose
I would do error handling in the method itself and give it a name like `checkWallAHKOpen`